'rep_handler' is an undefined name in ./examples/eventloop/echo.py

Undefined names can raise `NameError`at runtime.

flake8 testing of https://github.com/zeromq/pyzmq

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./examples/eventloop/echo.py:21:54: F821 undefined name 'rep_handler'
        ioloop.IOLoop.current().add_callback(partial(rep_handler, sock, events))
                                                     ^
```

--- a/examples/eventloop/echo.py
+++ b/examples/eventloop/echo.py
@@ -1,10 +1,9 @@
 #!/usr/bin/env python
 """A basic ZMQ echo server using the tornado eventloop
 
-without relying on tornado integration (see echostream).
+without relying on tornado integration (see echostream, echofuture).
 """
 
-from functools import partial
 import zmq
 from tornado import ioloop
 
@@ -14,11 +13,12 @@ def echo(sock, events):
         # not a read event
         return
     msg = sock.recv_multipart()
+    print(msg)
     sock.send_multipart(msg)
     # avoid starving due to edge-triggered event FD
     # if there is more than one read event waiting
     if sock.EVENTS & zmq.POLLIN:
-        ioloop.IOLoop.current().add_callback(partial(rep_handler, sock, events))
+        ioloop.IOLoop.current().add_callback(echo, sock, events)
 
 ctx = zmq.Context.instance()
 s = ctx.socket(zmq.ROUTER)

